internal/collector/pgadmin.go

```diff
@@ -29,7 +29,12 @@ func EnablePgAdminLogging(ctx context.Context, spec *v1beta1.InstrumentationSpec
 		"fsync":            true,
 	}
 
+	// https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/-/receiver/filelogreceiver#readme
 	otelConfig.Receivers["filelog/pgadmin"] = map[string]any{
+		// PgAdmin/gunicorn logs are rotated by python -- python tries to emit a
+		// log and if the file needs to rotate, it rotates first and then emits
+		// the log. The collector therefore only needs to watch the single active
+		// log for each component.
```

Comment:
Good comment, I might move it up a line so it looks like it applies to both `filelog/*` receivers -- I know you mention pgadmin and gunicorn, but I could imagine someone missing that since the comment is indented.